src/lib/i18next/resources.js

```diff
@@ -265,7 +272,7 @@ module.exports = {
       "Reopen": "ပြန်ဖွင့်မည်",
       "Report": "အစီရင်ခံစာ",
       "Reported By": "တင်သွင်းသူ",
-      "Reports": "အစီရင်ခံစာများ",
+      "Reports": "Reports",
```

Comment:
Why did this get overridden? If there are translations existing we shouldn't override right?

Reply:
Resolved!